src/test/java/examples/precisiondouble/DoubleSetInputScannerTest.java

```diff
@@ -0,0 +1,28 @@
+package examples.precisiondouble;
+
+import org.academiadecodigo.bootcamp.Prompt;
+import org.academiadecodigo.bootcamp.scanners.precisiondouble.DoubleSetInputScanner;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class DoubleSetInputScannerTest {
+
+    public static void main(String[] args) {
+
+        double firstValue = 225.25;
+        double secondValue = 403.14;
+
+        Prompt prompt = new Prompt(System.in, System.out);
+        Set<Double> options = new HashSet<>();
+        options.add(firstValue);
+        options.add(secondValue);
+
+        DoubleSetInputScanner scanner = new DoubleSetInputScanner(options);
+        scanner.setMessage("First: " + firstValue + " Second: " + secondValue + " ? ");
+        scanner.setError("Invalid value allow!");
```

Comment:
Can you please improve the messages on the example to make it clearer? I suggest something like `Insert one of + firstValue, secondValue :` as the scanner message and `Invalid value` as the error message, do you agree?

Reply:
Done!